json-endpoints/jetpack/class.jetpack-json-api-plugins-install-endpoint.php

```diff
@@ -38,6 +38,18 @@ protected function install() {
 		}
 
 		if ( ! $this->bulk && isset( $error ) ) {
+
+			if ( $error_code === 'download_failed' ) {
+				// For backwards compatibility: versions prior to 3.9 would return no_package instead of download_failed.
+				$error_code = 'no_package';
+			}
+
+			error_log( print_r( array(
```

Comment:
Sorry, but it looks like you meant to use `print_r` here with the second argument set to `true`.